subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -150,6 +150,8 @@ Notice how we specify the destination and name of the archive instead of an `int
 
 Each type of archive has its own task type, the most common ones being link:{groovyDslPath}/org.gradle.api.tasks.bundling.Zip.html[Zip], link:{groovyDslPath}/org.gradle.api.tasks.bundling.Tar.html[Tar] and link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar]. They all share most of the configuration options of `Copy`, including filtering and renaming.
 
+One notable difference between the `Copy` and `Sync` tasks and the archives is that starting with Gradle 6.0 duplicate files will not be allowed in archives anymore.
```

Comment:
We don't need to say `Gradle 6.0` here. Just state the difference, the docs are published for a version.

Reply:
It was like that in other sections, should I delete it from there as well?